### ISSUE TITLE
Be explicit about using classes from root namespace

### ISSUE THREAD

**Issue by dave1010** [cs-extended/fig-standards#4]
Can PSR-12 include whether or not to import classes from the root namespace? Eg 

```
namespace Foo;
use Exception;
throw new Exception();
```

or

```
namespace Foo;
throw new \Exception();
```

I prefer the latter, but there's not much in it.

**Comment by michaelcullum** [comment]
We could do this but again, I'm slightly worried about breaking BC here.

Could do it as a SHOULD, not MUST.

**Comment by michaelcullum** [comment]
I'm worried this could cause too much BC breaks and would rather not add SHOULDs. I'll defer this to the mailing list for further discussion.

**Comment by michaelcullum** [comment]
Closed in 6f20a32b9a9f3f4b9ce9b9b5d48fc30c50a2512b
